config/default.js

```diff
@@ -350,6 +352,10 @@ module.exports = {
 
   // Feature flags.
   //
+
+  enableFeatureDiscoTaar: false,
```

Comment:
I haven't seen any mention of a feature flag for that. Is there an issue for that?

Reply:
@muffinresearch mentioned to me to set this up with a flag. 

@muffinresearch should we still do this?